src/internal/src/win/AudioDestinationWin.cpp

```diff
@@ -9,9 +9,27 @@
 #include "LabSound/core/AudioIOCallback.h"
 #include "LabSound/extended/Logging.h"
 
+#include <rtaudio/RtAudio.h>
+
 namespace lab
 {
 
+static int
+NumDefaultOutputChannels() {
```

Comment:
ah-ha! same code from `AudioDestinationMac.cpp` -- better put this somewhere useful.